supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -136,20 +136,27 @@ If available, inform users of alternative capabilities and workarounds.
 
 [discrete]
 ===== Deprecation notice
+[subs="+quotes"]
 ----
-In <product> <release>, <name of capability> is deprecated. Bug fixes and support are provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.)
+In __<product_name> <release>__, __<name_of_capability_or_feature>__ is deprecated and is planned to be removed in the __<deprecation_timeline>__. Red Hat will provide bug fixes and support for this feature during the current release life cycle, but this feature will no longer receive enhancements and will be removed. As an alternative to __<name_of_capability_or_feature>__, you can use __<alternative_capability_or_feature_if_available>__ instead.
 ----
+[NOTE]
+====
+When citing deprecation timelines, refer to specific releases, such as __the next release__, only if that timeline is known to be accurate. Otherwise, use the phrase __a future release__ because it accounts for the possibility of changes to the planned deprecation timeline.
+====
+
 .Example deprecation notice doc text
 ----
-In Red Hat Openstack Platform (RHOSP) 14, the director graphical user interface is deprecated. Bug fixes and support is provided through the end of a future life cycle but Red Hat do not implement new feature enhancements.
+In Red Hat Openstack Platform (RHOSP) 14, the director graphical user interface is deprecated and is planned to be removed in a future release. Red Hat will provide bug fixes and support for this feature during the current release life cycle, but this feature will no longer receive enhancements and will be removed.
 ----
 
 [discrete]
 ===== Removal notice
+[subs="+quotes"]
 ----
-In <product> <release>, <name of capability> is removed. Bug fixes and support is provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.)
+In __<product_name> <current_release>__, __<name of capability or feature>__ is removed. Bug fixes and support are provided only through the end of the __<previous_release>__ life cycle. As an alternative to __<name_of_capability_or_feature>__, you can use __<alternative_capability_or_feature_if_available>__ instead. 
```

Comment:
Even though it might be grammatically correct, "is removed" doesn't sound right to my ears. Wouldn't "has been removed" be better?

Again, this was there before, but might as well consider now since we're fixing other grammatical issues in there. Appreciate thoughts on this one

Reply:
Good point, @bergerhoffer. I'll fix that.